sys-firmware/midisport-firmware/midisport-firmware-1.2.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+inherit autotools linux-info udev
+
+DESCRIPTION="Firmware for M-Audio/Midiman USB MIDI devices"
+HOMEPAGE="http://usb-midi-fw.sourceforge.net"
+SRC_URI="http://downloads.sourceforge.net/usb-midi-fw/${P}.tar.gz"
+
+LICENSE="Midisport"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="sys-apps/fxload
+	virtual/udev"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+CONFIG_CHECK="~SND_USB_AUDIO"
+
+src_prepare() {
+	eapply "${FILESDIR}"/${P}-configure.patch
```

Comment:
No. Please put those two patches in an array. By convention we call this array PATCHES. Then, within src_prepare(), you must call the epatch function and stringify the array like so: https://github.com/gentoo/gentoo/blob/335e115d3a96663ddced605dbb91b9eace608d4a/dev-java/colt/colt-1.2.0-r3.ebuild#L36.